app/redis_util/redis_connection.py

```diff
@@ -8,3 +8,6 @@
                 host=os.environ["REDIS_HOST"],
                 port=os.environ["REDIS_PORT"],
             )
```

Comment:
If you're going to use external name service for redis when we don't create, you can remove REDIS_HOST and REDIS_PORT from the configmap and get its host/port via k8s service discovery, similarly to how you are pinging aladdin demo. Doesn't need to be in this PR, but there is some minor inconsistency with how we are getting the redis endpoint vs how we are getting the demo svc endpoint.